### PR TITLE
Firefox 95 adds `ElementInternals` support for `.labels`, `.form`,  and `.setFormValue()`

### PR DESCRIPTION
**Pull request by hamishwillee**
FF95 adds support for the [ElementInternals.form](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/form), [ElementInternals.labels](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/labels), [ElementInternals.setFormValue()](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/setFormValue). See these bugs:
- [Custom elements: implement form attribute of ElementInternals](https://bugzilla.mozilla.org/show_bug.cgi?id=1556362)
- [Custom elements: implement labels for ElementInternals](https://bugzilla.mozilla.org/show_bug.cgi?id=1556373)
- [Custom elements: implement setFormValue() for ElementInternals](https://bugzilla.mozilla.org/show_bug.cgi?id=1556449)

### CODE REVIEW

**Comment by evilpie** [comment]
All of these are behind the flag [`dom.webcomponents.formAssociatedCustomElement.enabled`](https://searchfox.org/mozilla-central/search?q=dom.webcomponents.formAssociatedCustomElement.enabled) still.

**Comment by hamishwillee** [comment]
Thanks @evilpie - I did not know that or see it in bugzilla on quick scan. Will update this and docs on Friday.

File: api/ElementInternals.json [hunk 1]
```diff
@@ -112,7 +112,14 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.webcomponents.formAssociatedCustomElement.enabled",
+                  "value_to_set": "true"
+                }
+              ]
             },
             "firefox_android": {
               "version_added": false
```
Comment (hamishwillee) [inline api/ElementInternals.json:125]: Note, still false, because preference setting not supported on android.

**Comment by hamishwillee** [comment]
@ddbeck Ready for review

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**